app.js

```diff
@@ -284,21 +284,69 @@ window.onload = () => {
 };
 
 /* Validate form */
-function onSubmit(event) {
-  const validate = document.getElementById('email');
-  const errorMessage = document.getElementById('error');
-  const email = validate.value;
-
-  if (email.toLowerCase() !== email) {
-    event.preventDefault();
-    validate.classList.add('invalid');
-    errorMessage.classList.add('error');
-    errorMessage.innerText = 'Email should be lowercase!';
-  } else {
-    validate.classList.remove('invalid');
-    errorMessage.classList.remove('error');
+// selecting input elements
+const formName = document.getElementById('name');
+const formEmail = document.getElementById('email');
+const formComment = document.getElementById('textarea');
+
+// Store formData in localStorage
+function loadLocalStorage() {
+  const formData = JSON.parse(window.localStorage.getItem('formData'));
+  if (formData) {
+    formName.value = formData.name;
+    formEmail.value = formData.email;
+    formComment.value = formData.message;
+  }
```

Comment:
- [ ] There is a bug here making the `Textarea` value `undefined` when the page is reloaded. `formData.message` does not exist, according to your formData object, it should be formData.textarea. Let's fix that
<img width="393" alt="Screenshot 2022-11-19 at 1 10 18 PM" src="https://user-images.githubusercontent.com/40503704/202850187-55a78b91-2dc2-4223-a65e-79ba08b080f1.png">